### PR TITLE
Add fonts

### PR DESCRIPTION
**Pull request by Sedictious**

### CODE REVIEW

File: src/czml3/types.py [hunk 1]
```diff
@@ -32,6 +32,21 @@ def format_datetime_like(dt_object):
...
+    def __init__(self, *, font=None):
+        self._font = font
+
+    @property
+    def font(self):
+        """The font to use for the label."""
+        return self._font
+
+    def to_json(self):
+        return list(self._font)
```
Comment (astrojuanlu) [inline src/czml3/types.py]: Is this a list or a string?

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
According to the coverage report, there are several parts that are not tested:

https://codecov.io/gh/poliastro/czml3/pull/18/diff

Could we add some more tests, and perhaps expand also the `test_integration.py` if appropriate?

File: src/czml3/types.py (hunk 1 above)
Comment (Sedictious) [reply src/czml3/types.py]: My mistake! I also misunderstood how ``Font`` objects are parsed (I thought they're separate objects, while in reality they are simply strings). However, everything should work as expected now (also hit 100% coverage!)

Comment (astrojuanlu) [reply src/czml3/types.py]: Awesome! :grin:

**Review by astrojuanlu — APPROVED** [review]

**Comment by astrojuanlu** [comment]
Thanks @Sedictious! :tada:

**Pull request merged**